scraper/pyproject.toml

```diff
@@ -8,7 +8,7 @@ authors = [{ name = "Kiwix", email = "dev@kiwix.org" }]
 keywords = ["kiwix", "zim", "offline", "kolibri"]
 requires-python = ">=3.11"
 description = "Make ZIM file from Kolibri Channels"
-readme = "README.md"
+readme = "../README.md"
```

Comment:
What would be the difference with the main README ? 
The scraper cannot work without the `zimui` part, and the main README is already explaining all these dependencies.

Reply:
Well you wanted to segregate the two folders in the first place. It has the advantage of keeping things neat and self-contained. Now, this README creates an upper-level dependency (since zimui-dist can be passed as arg, it's different).
Also, we tend to have different ones because technologies are different and we want to encourage contributors to participate where they can.
It's a suggestion but single README is fine with me. Quality of said README is more important

Reply:
If you are fine with it, let's keep a single README for now, I'm afraid there will be lot's of repetitions otherwise.